components/movementsensor/gpsnmea/serial.go

```diff
@@ -39,8 +37,7 @@ type SerialNMEAMovementSensor struct {
 	lastCompassHeading movementsensor.LastCompassHeading
 	isClosed           bool
 
-	dev  io.ReadWriteCloser
-	path string
+	dev DataReader
```

Comment:
A reminder as much for me as for other reviewers: we're still calling `dev.Close()` down in `SerialNMEAMovementSensor.Close()`. Although this PR doesn't touch that function, we _are_ correctly closing the data reader.